mkosi.build

```diff
@@ -134,6 +134,28 @@ Kernel \r on an \m (\l)
 
 EOF
 
+if [[ -n "$IMAGE_ID" ]] ; then
```

Comment:
In the above, `[]` is used instead of `[[]]`. I am not familiar with which shell support `[[]]`, but maybe it is better to also use `[]` here, at least for consistency.